### ISSUE TITLE
we don't have a top level accessibility page?

### ISSUE THREAD

**Issue by tkimnguyen** [plone/ploneorg.core#209]
I think we should have Accessibility highlighted on the site somewhere, to refer to our noteworthy WCAG 2.0 compliance.

https://plone.org/accessibility

Should we bring over all high level features descriptions from Plone.com?

@polyester

**Comment by rioksane** [comment]
Yes we should. I added a page https://plone.org/accessibility with a short text that summarizes that.

**Comment by fulv** [comment]
There are several news items, in part related to awards and such, on the topic of accessibility, which we could link from this new page.

**Comment by skleinfeldt** [comment]
@rioksane should this have been closed, since the new page is very minimal and I thought the intention was to bring over all the plone.com text? Plus @fulv's idea is a good one.

I'm going to reopen, feel free to close this and open another ticket if you'd rather. (Or if you did that already!)

**Comment by rioksane** [comment]
I added accessibility text from https://plone.com/accessibility-info.html to https://plone.org/accessibility 

Not sure if everything is still valid

**Comment by rioksane** [comment]
page checked and fixed in plone.org so I will close the issue now as fixed.
